MSBuild.Obfuscar/build/MSBuild.Obfuscar.targets

```diff
@@ -14,11 +14,11 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup>
 
     <!-- ObfuscatorToolsPath -->
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference'">$(MSBuildThisFileDirectory)..\..\..\Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(NuGetPackageRoot)Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference'">$(MSBuildThisFileDirectory)..\..\..\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' == 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(NuGetPackageRoot)obfuscar\2.2.26\tools</ObfuscatorToolsPath>
 
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference'">$(MSBuildThisFileDirectory)..\..\Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
-    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(SolutionDir)packages\Obfuscar.2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference'">$(MSBuildThisFileDirectory)..\..\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
+    <ObfuscatorToolsPath Condition="'$(NuGetProjectStyle)' != 'PackageReference' And !Exists('$(ObfuscatorToolsPath)')">$(SolutionDir)packages\obfuscar\2.2.26\tools</ObfuscatorToolsPath>
```

Comment:
Not sure if this is correct. I think only the PackageReference style has another layout of the packages.

Reply:
https://www.nuget.org/packages/Obfuscar/